scipy/special/_support_alternative_backends.py

```diff
@@ -81,11 +81,30 @@ def __xlogy(x, y, *, xp=xp):
     return __xlogy
 
 
+def _chdtr(xp, spx):
```

Comment:
This is basically just a copy-paste of `_chdtrc`. let's get a few more of these under our belt before we try to factor out common code.